connection_test.go

```diff
@@ -868,3 +868,74 @@ func TestTosPriority(t *testing.T) {
 		require.NoError(t, err, "Failed to write to outbound conn")
 	})
 }
+
+func TestPeerStatusChangeClient(t *testing.T) {
+	sopts := testutils.NewOpts().NoRelay()
```

Comment:
can you add a comment above the `NoRelay` with the reason for why we don't enable the relay